.github/workflows/ci.yaml

```diff
@@ -1,6 +1,9 @@
 name: ci
 
-on: [pull_request] # yamllint disable-line rule:truthy
+on:
+  pull_request:
+    paths-ignore:
```

Comment:
Documentation for the `paths-ignore` configuration is [here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore)